browser-switch/build.gradle

```diff
@@ -36,12 +36,6 @@ dependencies {
     testImplementation deps.mockitoCore
     testImplementation deps.jsonassert
     testImplementation deps.robolectric
-
-    // TODO: remove powermock
-    testImplementation 'org.powermock:powermock-module-junit4:2.0.9'
```

Comment:
This isn't directly related to Android 14 - but powermock was causing errors preventing me from running the test suite and we intended to remove it anyways

Reply:
🔥 Yeah this is awesome. This was also additional motivation for moving to Kotlin. Mockk is a lot more dev friendly.